src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -156,43 +156,16 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchCurrentUser(currentUser);
     }
 
-    componentDidUpdate({ annotatorState: prevAnnotatorState, match: prevMatch }: Props): void {
-        const { annotation: prevAnnotation } = prevAnnotatorState;
-        const {
-            annotatorState: { annotation },
-            match,
-        } = this.props;
+    componentDidUpdate({ match: prevMatch }: Props): void {
+        const { match } = this.props;
         const prevFileVersionId = getProp(prevMatch, 'params.fileVersionId');
         const fileVersionId = getProp(match, 'params.fileVersionId');
 
-        if (prevAnnotation !== annotation) {
-            this.addAnnotation();
-        }
-
         if (prevFileVersionId !== fileVersionId) {
             this.updateActiveVersion();
```

Comment:
Will this need to move up as well or no?

Reply:
Yeah, all his friends have moved up... but I think this should actually stay here, at least for now. `feedItems` is where we have the full version objects and moving it up would require the fetching of versions to be done higher up. Maybe we can revisit this as part of the future refactor of having ContentPreview load a previous versions directly